doc/database-model.md

```diff
@@ -0,0 +1,61 @@
+
+# Database Entity Relationship Diagram
+
+```mermaid
+erDiagram
+    USER {
+        String _id PK
+        String name
+        String googleID
+        String address
+        ObjectId[] dogs
+    }
+    DOG {
+        String _id PK
+        String name
+        String breed
+        Number age
+        Date birth
+        Number weight
+        Boolean neutered
+        String medication
+        Number medicationPurchaseFrequency
+        String vetCardImageUrl
+        String qrChipImageUrl
+        String[] allergies
+        String[] illness
+        ObjectId[] foods
+        ObjectId[] medical_history
```

Comment:
if you are using camel case, use it in all fields

Reply:
then the table will be called medicalhistory. if there's a better name for it or only medical i'm open to suggestions